api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2969,4 +2969,101 @@ public void saveOrder_shouldPassIfAnKnownDrugOrderForTheSameDrugFormulationSpeci
 		orderService.saveOrder(order, orderContext);
 		assertNotNull(orderService.getOrder(order.getOrderId()));
 	}
+	
```

Comment:
I think you are missing some tests, see the acceptance criteria for the ticket and make sure all those scenario have been or already exist